Trackovid19-web/src/app/screens/main/main.component.ts

```diff
@@ -97,10 +97,10 @@ export class MainComponent implements OnInit, OnDestroy {
         this.user = { ...user };
         this.loadState();
 
-        // uncomment this and change to the correct property when #120 is done
-        //if (this.hasUserOutdatedStatus(user.updated_at)) {
-        //this.router.navigate(['/dashboard', 'change-state-step1']);
-        //}
+        const latestUpdateTime = user.latest_status.timestamp;
```

Comment:
Can you consider removing this `const` and calling the method directly with the value (`user.latest_status.timestamp`), which will prevent this memory allocation for this variable, which is used only once in the code block?

Reply:
+1 to this suggestion

Reply:
Sure, I will remove it.

 I just added it for readability, and since this will only run once (loading user data) I didn't see a big deal/problem.

Reply:
@pfortuna and @miguelleite done